iron-ajax.html

```diff
@@ -353,13 +353,15 @@
         contentType = 'application/x-www-form-urlencoded';
       }
       if (contentType) {
-        headers['Content-Type'] = contentType;
+        headers['content-type'] = contentType;
       }
       var header;
 
       if (this.headers instanceof Object) {
         for (header in this.headers) {
-          headers[header] = this.headers[header].toString();
+          // Normalize headers in lower case to make it easier for iron-request
+          // to reason about them.
+          headers[header.toLowerCase()] = this.headers[header].toString();
```

Comment:
I wanted to walk the headers as few times as possible, but I didn't think about folks using iron-request directly. I'll put together a PR that moves the lowercasing to iron-request.